earth_orography/README.md

```diff
@@ -0,0 +1,34 @@
+# Earth orography (ETOPO2022 Dataset)
+
+
+## Overview 
+
+This artifact provides the 30 and 60 arc-second ice-surface orography information 
+in NetCDF format. 
+- ETOPO_2022_v1_30s_N90W180_surface.nc (30 arc-second, ice surface, 1.5GB)
+- ETOPO_2022_v1_60s_N90W180_surface.nc (60 arc-second, ice surface, 456MB)
```

Comment:
Give the relative sizes, I suggest we split this into two artifacts: one that is automatically downloaded (60 arcseconds), and one that needs to be downloaded manually. To do so, you can just call `create_artifact_guided` twice (with `append` the second time). Have a look at SST and SIC for an exmaple

Reply:
@Sbozzolo Updated to set up 2 artifacts, one as an auto download (data has been uploaded to cluster with the `Overrides.toml` file updated. The lower resolution data is available via Caltech (static link) box or the HPC storage.

Reply:
E.g. output in a test folder: 
```% julia --project -e 'using LazyArtifacts; println(artifact"earth_orography_60arcseconds")'
     Failure artifact: earth_orography_60arcseconds
  Downloaded artifact: earth_orography_60arcseconds
```

Reply:
I don't know why it always fails the first time